### ISSUE TITLE
Very poor performance on second level of Waterway

### ISSUE THREAD

**Issue by PIlin** [PIlin/NXEngine-iOS#16]
~15 FPS on iPad 2

Also, there were a few places (like where Prof. Booster falling) where FPS was low.

**Comment by PIlin** [comment]
Poor performance due to render to texture on each frame. It seems, this operation is very slow on ios devices. It's better on release configuration, almost 50 fps, but there are random little freezes anyway.

**Comment by octopuserectus** [comment]
Same on 4S 5.1.1
This level is almost unplayable. What do you mean by «It's better on release configuration»?

**Comment by PIlin** [comment]
There are debug and release configuration of build. When I first discovered this problem, debug version was installed on my device. Wehn I checked last, it was release configuration on my device and it was as I described in my last comment. But, probably, I made a mistake - there are two waterway levels and, maybe, I've checked wrong one.

Also, it's possible to beat level. I even made this when almost everything was covered by black squares.
